### ISSUE TITLE
Nuvoton NCT6771F High temperatures

### ISSUE THREAD

**Issue by Stealhelm** [openhardwaremonitor/openhardwaremonitor#1097]
Getting temperatures on the #2 sensor at 110 - 125°C. The other two are at a cool ~40°-ish. CPU is at 42°C with dips into the mid 30s. Is this common or am I experiencing an bug?

System Information:
CPU - Intel I5 750, 4.0ghz
Motherboard - AsRock P55 Deluxe3
OS - 64-bit Windows 10 Pro, 10.0.16299, Build 16299

PS: Do apologize if this isn't an issue just seems really strange - I hope you can understand my concerns.

**Comment by DarthDestroyer** [comment]
if you could upload a report, it could be looked at! ;)

**Comment by Stealhelm** [comment]
> if you could upload a report, it could be looked at! ;)

Clocked down the CPU with no luck.
[OpenHardwareMonitor.Report.txt](https://github.com/openhardwaremonitor/openhardwaremonitor/files/1886512/OpenHardwareMonitor.Report.txt)

**Comment by Stealhelm** [comment]
> atleast your version is outdated > http://openhardwaremonitor.org/files/openhardwaremonitor-v0.8.0.3-alpha.zip

Oh, sorry, I thought I downloaded the latest version.

> as i remember correct, there was a similar issue already?! nevermind! Could you try the latest Version?

Sure thing - here's the report:
[OpenHardwareMonitor.Report.txt](https://github.com/openhardwaremonitor/openhardwaremonitor/files/1886561/OpenHardwareMonitor.Report.txt)

**Comment by DarthDestroyer** [comment]
The MoBo Sensors are sometimes a little bit weird... As long as your CPU Temp itself doesnt do crazy things, it isnt dangerous!

How do you know, that Temp#2 is a CPU related Sensor?

**Comment by Stealhelm** [comment]
> How do you know, that Temp#2 is a CPU related Sensor?

Oh, I don't know that. I just assumed that seeing as the only thing I've changed was my OC it could've been interesting to know. Oh well. If it isn't dangerous I wont bother.

Thanks for the information - greatly appriciated! :)

**Comment by pimentelleo** [comment]
I am facing the same, just downloaded the latest version. 
I was playing Astroneer, then the things goina crazy about 117°C

**Comment by kayhen** [comment]
there are three NCT6779D with high temperature
79C and 93C and one of them is at 22C
But I don't know if this temperature is normal.

**Comment by borovinskiy** [comment]
For my NCT6779D temp also incorrect:
CPU6: -23 C.
CPU3,4: 90+ C
[OpenHardwareMonitor.Report.txt](https://github.com/openhardwaremonitor/openhardwaremonitor/files/7514016/OpenHardwareMonitor.Report.txt)

**Comment by namxam** [comment]
Yeah, I am seeing the same behaviour with the latest 0.9.7 release. Sensor 2 seems to be okay, the rest is showing values beyong 115˚C.

Is there anything we could provide to fix the issue? I have a few machines with the same config showing similar issues.

**Comment by STARasGAMES** [comment]
Made me panic for a few minutes :)
![image](https://user-images.githubusercontent.com/23558898/144381842-6d9a0767-991d-4414-a6d5-18f36796446c.png)

Glad to know it's not dangerous. But would be cool if it would be "cool"... XD

**Comment by handsomemans-rich** [comment]
![X99 Nuvoton NCT6779D overheat](https://github.com/user-attachments/assets/ee95928a-c7b5-4d0b-86d2-66510227e12a)

Maiherboard Dual Xeon X99 Nuvoton NCT6779D overheat???
BSOD many time.

[Uploading OpenHardwareMonitor.Report.txt…]()


![X99 Nuvoton NCT6779D overheat](https://github.com/user-attachments/assets/6d193735-3440-4124-b6b1-9f4fc3ac54d7)

**Comment by DarthDestroyer** [comment]
Check the Documentation of your MoBo where the Temp-Sensors are located!
HWMonitor and OHM are "only" reading what the Mainboard itself delivers - as you see, two Tools are measuring nearly the same Temp!

Maybe its located under the NVME or under the CPU or maybe at the Power-Parts - these are the Hotspots, i would say...
It depends! Look at the Docs or Manual where the Sensors are...

For sure, Temp Problems can lead to a BSOD - But that isnt the Fault from OHM
